tests/deployments_test.go

```diff
@@ -101,3 +101,63 @@ func deploymentSubTest(t *testing.T, mode api.DeploymentMode, engine api.Storage
 
 	return nil
 }
+
+// test setup containing multiple deployments
+func TestMultiDeployment1(t *testing.T) {
+	longOrSkip(t)
+
+	k8sNameSpace := getNamespace(t)
+	k8sClient := mustNewKubeClient(t)
+	deploymentClient := kubeArangoClient.MustNewInCluster()
+
+	// Prepare deployment config
+	deploymentTemplate1 := newDeployment("test-multidep1-1-" + uniuri.NewLen(4))
+	deploymentTemplate1.Spec.Mode = api.NewMode(api.DeploymentModeCluster)
+	deploymentTemplate1.Spec.StorageEngine = api.NewStorageEngine(api.StorageEngineRocksDB)
+	deploymentTemplate1.Spec.TLS = api.TLSSpec{}                        // should auto-generate cert
+	deploymentTemplate1.Spec.SetDefaults(deploymentTemplate1.GetName()) // this must be last
+
+	deploymentTemplate2 := newDeployment("test-multidep1-2-" + uniuri.NewLen(4))
+	deploymentTemplate2.Spec.Mode = api.NewMode(api.DeploymentModeSingle)
+	deploymentTemplate2.Spec.StorageEngine = api.NewStorageEngine(api.StorageEngineMMFiles)
+	deploymentTemplate2.Spec.TLS = api.TLSSpec{}                        // should auto-generate cert
+	deploymentTemplate2.Spec.SetDefaults(deploymentTemplate2.GetName()) // this must be last
+
+	// Create deployment
+	deployment1, err := deploymentClient.DatabaseV1alpha().ArangoDeployments(k8sNameSpace).Create(deploymentTemplate1)
+	if err != nil {
+		t.Fatalf("Create deployment failed: %v", err)
+	}
+
+	deployment2, err := deploymentClient.DatabaseV2alpha().ArangoDeployments(k8sNameSpace).Create(deploymentTemplate2)
+	if err != nil {
+		t.Fatalf("Create deployment failed: %v", err)
+	}
+
+	// Wait for deployment to be ready
+	deployment1, err = waitUntilDeployment(deploymentClient, deploymentTemplate1.GetName(), k8sNameSpace, deploymentIsReady())
+	if err != nil {
+		t.Fatalf("Deployment not running in time: %v", err)
+	}
+
+	deployment2, err = waitUntilDeployment(deploymentClient, deploymentTemplate2.GetName(), k8sNameSpace, deploymentIsReady())
+	if err != nil {
+		t.Fatalf("Deployment not running in time: %v", err)
+	}
+
+	// Create a database client
+	ctx := context.Background()
+
+	DBClient1 := mustNewArangodDatabaseClient(ctx, k8sClient, deployment1, t)
+	if err := waitUntilArangoDeploymentHealthy(deployment1, DBClient1, k8sClient, ""); err != nil {
+		t.Fatalf("Deployment not healthy in time: %v", err)
+	}
+	DBClient2 := mustNewArangodDatabaseClient(ctx, k8sClient, deployment2, t)
+	if err := waitUntilArangoDeploymentHealthy(deployment2, DBClient2, k8sClient, ""); err != nil {
+		t.Fatalf("Deployment not healthy in time: %v", err)
+	}
+
```

Comment:
To make sure that we can talk to individual deployments, I'll like to see an additional test here.
- Create a collection in deployment 1 (named `col1`)
- Create a collection in deployment 2 (named `col2`)
- Check that deployment 1 has a collection named `col1` but not `col2`
- Check that deployment 2 has a collection named `col2` but not `col1`